src/fidgets/ui/IFrame.tsx

```
@@ -312,6 +264,114 @@ const IFrame: React.FC<FidgetArgs<IFrameFidgetSettings>> = ({
+    // Otherwise wait for load event
+    if (iframe.contentWindow) {
+      setupHandler();
+    } else {
+      iframe.addEventListener('load', setupHandler, { once: true });
+    }
+  }, [contextForEmbedded, transformedUrl, rawSdkInstance, authenticatorManager]);
+  
+  // Cleanup on unmount
+  useEffect(() => {
+    return () => {
+      comlinkCleanups.current.forEach(cleanup => cleanup());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)